src/Orc.ProjectManagement/Managers/ProjectManager.cs

```diff
@@ -317,6 +325,7 @@ await ProjectActivatedAsync
             }
         }
 
+        [Time]
```

Comment:
I am not entirely sure how this will behave when overridden. I think this can be removed and put into the actual implementing class.